examples/udp-ssdp-search/main.c

```diff
@@ -8,11 +8,24 @@ static const char *s_ssdp_url = "udp://239.255.255.250:1900";
 static void fn(struct mg_connection *c, int ev, void *ev_data, void *fn_data) {
   MG_DEBUG(("%p got event: %d %p %p", c, ev, ev_data, fn_data));
   if (ev == MG_EV_OPEN) {
-    c->is_hexdumping = 1;
+//    c->is_hexdumping = 1;
   } else if (ev == MG_EV_RESOLVE) {
     // c->rem gets populated with multicast address. Store it in c->label
     memcpy(c->label, &c->rem, sizeof(c->rem));
   } else if (ev == MG_EV_READ) {
+    MG_INFO(("Got a response"));
+    struct mg_http_message hm;
+    if(mg_http_parse((const char *)c->recv.buf, c->recv.len, &hm) > 0){
```

Comment:
Please format the code using clang-format utility.
Mongoose repo has .clang-format file in the root, which will be used by the utility to choose the right style.
You might add a plugin to your editor to do that automatically on save: if you have a .clang-format file in your $HOME, it'll use your personal settings, in case if a repo/directory you're working on, does not have that file.